docs/topic_guide/channel_response.rst

```diff
@@ -0,0 +1,148 @@
+.. _sunkit-instruments-topic-guide-channel-response:
+
+************************************
+A Vocabulary for Instrument Response
+************************************
+
+This topic guide provides a vocabulary for response functions for imaging instruments.
+The goal of providing a single vocabulary for instrument response calculations is to provide a specification for a common interface that can be used by multiple instruments.
+This reduces the amount of effort needed to develop analysis software for new instruments and enables cross-instrument comparisons as upstream packages and users can program against a single interface for these response calculations.
+An abstract implementation of this vocabulary is provided in this package.
+
+Temperature Response
+--------------------
+
+The temperature response describes the instrument sensitivity as a function of temperature.
+It is a useful quantity when performing thermal analysis of imaging data, such as a differential emission measure or filter ratio analysis.
+The temperature response is defined as,
+
+.. math::
+
+    K(T) = \int\mathrm{d}\lambda\,R(\lambda)S(\lambda,T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1} \,\mathrm{cm}^5]
+
+It has a physical type of data number (DN) per pixel per unit time per unit emission measure.
+Note that the temperature response is a function of *both* the instrument properties as well as the atomic physics of the emitting source.
+The temperature response is related to the observed intensity in a given pixel by,
+
+.. math::
+
+    I = \int\mathrm{d}T\,K(T)\mathrm{DEM}(T)\quad[\mathrm{DN}\,\mathrm{pixel}^{-1}\,\mathrm{s}^{-1}],
+
+where :math:`\mathrm{DEM}(T)=n^2\frac{dh}{dT}` is the line-of-sight *differential* emission measure distribution in a given pixel and is typically express in units of :math:`\mathrm{cm}^{-5}\,\mathrm{K}^{-1}`. 
+
+Source Spectra
+--------------
+
+The source spectra describes how a source is emitting as a function of wavelength and temperature.
```

Comment:
This is close to the spectral radiance (as defined by wikipedia)

Reply:
It's close modulo a factor of cm$^{-5}$, i.e. the emission measure. I'm hesitant to call it the spectral radiance since the product of this and the emission measure would actually be the spectral radiance in the appropriate units.

Reply:
I've added a note in the description that this may also be referred to as the spectral radiance per unit emission measure.

Reply:
Ah yes, I was reading too quickly. Spectral radiance per unit emission measure sounds good to me.